playbooks/tests/tasks/integration.yml

```diff
@@ -96,6 +96,14 @@
     size: 2
   when: cinder.enabled|bool
 
+- name: check volume available before attach
+  shell: . /root/stackrc && {{ item }}
+  args:
+    executable: /bin/bash
+  with_items:
```

Comment:
We should avoid indefinite loop. Also, might be good to increase sleep to 5. Should fail if not available in 30 secs.

Probably good to using retires option instead of while loop